torchvision/datasets/fer2013.py

```diff
@@ -23,8 +23,8 @@ class FER2013(VisionDataset):
     """
 
     _RESOURCES = {
-        "train": ("train.csv", "3f0dfb3d3fd99c811a1299cb947e3131"),
-        "test": ("test.csv", "b02c2298636a634e8c2faabbf3ea9a23"),
+        "train": ("icml_face_data.csv", "b114b9e04e6949e5fe8b6a98b3892b1d"),
+        "test": ("icml_face_data.csv", "b114b9e04e6949e5fe8b6a98b3892b1d"),
```

Comment:
Thank you for the PR @NSalberg .

The feature request looks reasonable, however I think we should avoid forcing the existence of `icml_face_data.csv` because that will potentially break existing users who only have downloaded `train.csv` and `test.csv`.

Perhaps we should instead check the existence of `icml_face_data.csv`  and use it if it exists, and otherwise fall back to `train.csv` and `test.csv`?